packages/tokens/src/tokens.scss

```diff
@@ -145,8 +145,7 @@
   --button-small-letter: 0.08em;
   --text-caption: var(--font-weight-regular) 12px/20px var(--font-family);
   --text-caption-letter: 0.004em;
-  --text-caption-focused: var(--font-weight-regular) 12px/20px
-    var(--font-family);
+  --text-caption-focused: var(--font-weight-bold) 12px/20px var(--font-family);
```

Comment:
**Question**🤔:  You need to also update the [docs](https://juntossomosmais.github.io/atomium/pr-preview/pr-263/?path=/docs/tokens-typography--docs)
<img width="891" alt="Captura de Tela 2023-09-14 às 13 13 23" src="https://github.com/juntossomosmais/atomium/assets/6757777/ca22ec14-7e25-4192-a0b6-963e0ac45cf0">

Reply:
@schirrel thank for the reminder! https://github.com/juntossomosmais/atomium/pull/261/commits/feadbfc18a7e4bee537f8c77b65fea7c1f941994